docs/release-notes/deprecate-priority-scheduler.rst

```diff
@@ -0,0 +1,6 @@
+:orphan:
+
+**Deprecations**
+
+-  Kubernetes Scheduling: Due to limited usage, we are removing support for the priority scheduler
+   with preemption for Kubernetes. Users should transition to the default scheduler.
```

Comment:
nit: 
`Kubernetes Scheduling: Due to limited usage, we are discontinuing support for the priority scheduler with preemption for Kubernetes. Users are advised to transition to the default scheduler.`